nemo/collections/llm/gpt/model/base.py

```diff
@@ -25,6 +25,9 @@
 
     from nemo.collections.common.tokenizers.tokenizer_spec import TokenizerSpec
 
+TRAIN_LOSS_REDUCTION = MaskedTokenLossReduction()
+VALIDATION_LOSS_REDUCTION = MaskedTokenLossReduction(validation_step=True)
```

Comment:
Let's comment that this is a temporary WAR.

Reply:
Or remove the globals.

Reply:
Updated.